app/src/main/java/net/gsantner/markor/frontend/textview/TextViewUtils.java

```diff
@@ -713,39 +713,4 @@ public static Boolean isImeOpen(final View view) {
         }
         return null; // Uncertain
     }
-
-    public interface MTextWatcher extends TextWatcher {
-        // Classes inheriting from this class will be disabled in withAutoFormatDisabled
-    }
-
-    public static void withAutoFormatDisabled(final Editable editable, final GsCallback.a0 callback) {
```

Comment:
Unnecessary and non-performant code. Switching back to the old method for simplicity.